packages/expo-video/src/VideoView.web.tsx

```diff
@@ -22,6 +23,18 @@ class VideoPlayerWeb implements VideoPlayer {
   get isMuted(): boolean {
     return this._isMuted;
   }
+
+  set volume(value: number) {
+    this.mountedVideos.forEach((video) => {
+      video.volume = value;
+    });
+    this._volume = value;
+  }
+
+  get volume(): number {
+    return this._volume;
```

Comment:
We'll probably need to listen for changes too. The volume might be changed by the user using the player controls.